src/probnum/diffeq/odefiltsmooth/odefiltsmooth.py

```diff
@@ -27,7 +27,8 @@ def probsolve_ivp(
     ivp,
     method="ekf0",
     which_prior="ibm1",
-    tol=None,
+    atol=None,
```

Comment:
At some point in the future, yes. For now, let's discuss this in a separate issue I guess. I expect this will boil down to whether we are confident enough in our current adaptive step implementation to make this the default.